content/general/functions.textile

```diff
@@ -103,31 +91,28 @@ h4. Example @channel.message@ JSON payload
    ]
 }
 ```
-
-Please note that if you are planning to receive messages via Ably Functions, it is theoretically very easy to exceed the "transport rate limits":#transport we impose on Reactor Function to prevent DoS attacks against your own servers. We recommend you consider "message queues":/general/queues instead as they are far more scalable.
+Please note that if you are planning to receive messages via Ably Functions, it is theoretically very easy to exceed the "transport rate limits":https://support.ably.io/support/solutions/articles/3000053845-do-you-have-any-connection-message-rate-or-other-limits-on-accounts we impose on Reactor Function to prevent DoS attacks against your own servers. We recommend you consider "message queues":/general/queues instead for a more scalable solution, or "firehose":/general/firehose.
```

Comment:
Well given Reactor Functions push messages to serverless architecture, it feels wrong to say "DoS attacks against your own servers".  Can we use something more appropriate like a "DoS attack against your endpoints"